nss-day-cohort-51/nutshell-vampire-brood
# Log out button added to Navbar; Component wireframe add to public/images

# Description: Added a working **Log out** button to the Navbar

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions

1. In your web browser, visit the application home page and login if not already logged in
2. Click the button **Log out** in the navigation bar
3. You will taken back to the login page
4. Open **DevTool** -> **Application** tab. Under **Storage**, click **Session Storage**. You should not see a nutshell_user key
5. Test complete

# Checklist:
